docs/content/misc/contribute.ngdoc

```diff
@@ -61,7 +61,8 @@ To build AngularJS, you clone the source code repository and use Grunt to genera
 minified AngularJS files:
 
 ```shell
-# Clone your Github repository:
+# Clone your Github repository(Note: Do not go with `sudo` user when cloning into this repository 
```

Comment:
Yes i totally agree with you @Narretz , but won't it be a before hand note to tell the users that using sudo might not be good always, if in case they use it to clone the repo and are not even aware of what exactly it does?